freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/health/HealthDetailsFreeIpaResponse.java

```diff
@@ -0,0 +1,87 @@
+package com.sequenceiq.freeipa.api.v1.freeipa.stack.model.health;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.doc.FreeIpaModelDescriptions;
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.model.common.Status;
+import com.sequenceiq.service.api.doc.ModelDescriptions;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel("HealthDetailsFreeIpaV1Response")
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class HealthDetailsFreeIpaResponse {
+    @NotNull
+    @ApiModelProperty(value = ModelDescriptions.ENVIRONMENT_CRN, required = true)
+    private String environmentCrn;
+
+    @NotNull
+    @ApiModelProperty(value = FreeIpaModelDescriptions.FREEIPA_NAME, required = true)
+    private String name;
+
+    @NotNull
+    private String crn;
+
+    @NotNull
+    private List<NodeHealthDetailsFreeIpaResponse> nodeHealthDetailsFreeIpaResponses;
```

Comment:
could you please shortener the name? It is very difficult to handle on the client side (e.g.: any cli)

Reply:
changed.

Reply:
done